ADAL/src/public/ADAuthenticationParameters.h

```diff
@@ -86,4 +87,15 @@ typedef void (^ADParametersCompletion)(ADAuthenticationParameters * _Nullable pa
 + (void)parametersFromResourceUrl:(nonnull NSURL *)resourceUrl
                   completionBlock:(nonnull ADParametersCompletion)completionBlock;
 
+
+/*!
+    Generates default WKWebviewConfiguration with "PkeyAuth/1.0" keyword appended to the UserAgent String.
+    The user can initialize an embedded webview with the default configuration returned from this API to enable PKeyAuth challenge.
+ */
+#if TARGET_OS_IPHONE && !MSID_EXCLUDE_WEBKIT
```

Comment:
I think it should be available for both iOS and macOS, no?
Also, we shouldn't leak internal macro to public space (MSID_EXCLUDE_WEBKIT).